worker/src/d1/mod.rs

```diff
@@ -372,6 +384,18 @@ impl D1Result {
             Ok(Vec::new())
         }
     }
+
+    /// Return the meta data in this result.
+    ///
+    /// Returns `None` if it's not exists.
```

Comment:
I'm actually not sure when this would not be included in the result.
```suggestion
    /// Returns `None` if `meta` field is not populated.
```